plugins/coreboot/fu-plugin-coreboot.c

```diff
@@ -0,0 +1,217 @@
+/*
+ * Copyright (C) 2019 9elements Agency GmbH <patrick.rudolph@9elements.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#include "config.h"
+
+#include <string.h>
+#include <stdio.h>
+#include <glib.h>
+#include <glib/gstdio.h>
+
+#include "fu-plugin-vfuncs.h"
+#include "fu-device-metadata.h"
+#include "fu-device-private.h"
+#include "fu-plugin-coreboot.h"
+
+#define ARRAY_SIZE(a) (sizeof(a) / sizeof((a)[0]))
+
+void
+fu_plugin_init (FuPlugin *plugin)
+{
+	/* make sure that flashrom plugin is ready to receive devices */
```

Comment:
Then I think you need some ordering stuff to make sure this inits later.

Reply:
removed, will be added with libflashrom support